packages/viewer/src/scenes/cameraRenderResult.ts

```diff
@@ -0,0 +1,40 @@
+import { Frame } from '../types';
+import { StreamApi } from '@vertexvis/stream-api';
+import { Result } from '../stream/result';
+import { StreamApiEventDispatcher } from '../stream/dispatcher';
+
+export interface RenderResultIds {
+  animationId?: string;
+  correlationId?: string;
+}
+
+export class RenderResult implements Result {
+  public data = undefined;
+  public onAnimationCompleted: StreamApiEventDispatcher<string | undefined>;
+  public onFrameReceived: StreamApiEventDispatcher<Frame.Frame | undefined>;
```

Comment:
These signatures still have `undefined` in them.

Reply:
Hmm, apparently I wound up with an outdated version of that file being pushed up. I'll correct that